e2e-tests/functions

```diff
@@ -13,8 +13,14 @@ deploy_operator() {
 		"$(printf 'select(documentIndex==1).spec.template.spec.containers[0].image="%s"' "${IMAGE}")" \
 		"${DEPLOY_DIR}/operator.yaml" \
 		| kubectl -n "${NAMESPACE}" apply -f -
+}
 
+deploy_cluster_secrets() {
+	local secrets_to_apply=${1}
 	kubectl -n "${NAMESPACE}" apply -f "${TESTS_CONFIG_DIR}/secrets.yaml"
+	if [[ ${secrets_to_apply} == "all" ]]; then
+		kubectl -n "${NAMESPACE}" apply -f "${TESTS_CONFIG_DIR}/ssl-secret.yaml"
```

Comment:
it is a bit unclear the diff between deploy_cluster_secrets without param and deploy_cluster_secrets all. We need to check the code to understand it. 
I believe it will be better to use more understandable key name. Something like withssl withoutssl. 
@cap1984 what do you think?

Reply:
agree